### DISCUSSION TITLE
Welcome to KerasFuzzy Discussions!

### DISCUSSION THREAD

**Discussion by kenoma** [kenoma/KerasFuzzy#3] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by navya-git-sw** [comment]
Hi @kenoma , I hope you are doing well. In example codes i.e.  [case5_iris_dataset.py](https://github.com/kenoma/KerasFuzzy/blob/master/KerasFuzzy/examples/case5_iris_dataset.py), [case6_breast_cancer_dataset.py](https://github.com/kenoma/KerasFuzzy/blob/master/KerasFuzzy/examples/case6_breast_cancer_dataset.py) are adaptive neuro fuzzy implementation models or only fuzzy implementation models? ANFIS implemented in those codes? Could you please clarify I am new to this ANFIS algorithm.

**Reply by kenoma** [reply]
Hello, it's just components to build fuzzy systems. 
If you'd like to build ANFIS you should combine FuzzyLayer, an normalization layer, say, softmax layer and DefuzzyLayer.

**Reply by navya-git-sw** [reply]
Thank you for responding. 
Okay. I have to add like in [case2_neuro_fuzzy.py](https://github.com/kenoma/KerasFuzzy/blob/master/KerasFuzzy/examples/case2_neuro_fuzzy.py) right?

**Reply by kenoma** [reply]
Guess, more like this:
```python
inputs = keras.Input(....)
fz = FuzzyLayer(2)(inputs)
fz = tf.keras.layers.Softmax()(fz)
fz = DefuzzyLayer(10)(fz)

```
